bot/constants.py

```diff
@@ -80,3 +80,6 @@ class Roles(NamedTuple):
     ERROR_REPLIES = bot_replies["ERROR_REPLIES"]
     POSITIVE_REPLIES = bot_replies["POSITIVE_REPLIES"]
     NEGATIVE_REPLIES = bot_replies["NEGATIVE_REPLIES"]
+
+# Minecraft Server
+MC_SERVER_ADDRESS = "mc.gurkult.com"
```

Comment:
```suggestion
class Minecraft(NamedTuple):
    server_address = "mc.gurkult.com"
```
I think making this a NamedTuple, like the rest of the constants, would be nice. The command above could also be removed if you do this.

(Just remember to change the references to the address if you do this.)